README.md

```diff
@@ -8,59 +8,44 @@ What this project creates is a [composition](https://storybook.js.org/docs/react
 
 Then we use a pre-built manager UI, configured to compose the skeleton, in order to get the "full" Storybook experience with the skeleton-built preview.
 
-## Example projects
-
-The skeleton is setup to run stories from a collection of example real-world projects of varying complexity.
-
-Currently we have two examples running:
-
-- `template` -- the basic set of stories that are installed by the cli when you init a SB project. This project has 8 stories across 3 components.
-
-- `design-system` -- the stories from the [Storybook Design System](https://github.com/storybookjs/design-system). This project has 181 stories across 42 components.
-
-We may add more projects to the skeleton in the future but it is also relatively simple to configure it to run with other projects.
-
-### Configuring examples
-
-To configure an example named `$EXAMPLE`, you must provide 3 files:
-
-- `stories-json/$EXAMPLE.stories.json` -- a `stories.json` as generated by a Storybook build using `6.2.0-alpha.30` or above. _ NOTE: currently you may need to hand-edit the `fileNames` in this file to match up with the file names expected by `require.context`. _
-- `src/$PROJECT-entry.js` - a file that calls `configure` from `./storybook` with a `require.context` for the example's set of story files, the contents of the `stories.json` and (optionally) an imported `.storybook/preview.js`.
-
-_NOTE:_ if the `require.context` is `lazy`, the CSF files will be loaded on-demand.
-
 ## Running the skeleton
 
-Currently the process to build and run the skeleton is as follows:
+The process to build and run the skeleton is as follows:
 
-0. If running the design system, check [the code](https://github.com/storybookjs/design-system) out and make a symlink from `./design-system` to it.
+1. Check out the skeleton code on your machine and symlink inside your project at `skeleton`
 
-1. Build the webpack project into dist, with the `PROJECT` environment variable set:
+2. Ensure you have `webpack`, `webpack-cli` and `webpack-dev-server` installed.
 
-```
-export PROJECT=design-system; yarn start-wps
-```
+3. Add a `skeletonWebpackConfig` field to `.storybook/main.js` with the minimal webpack config you need to compile your stories/components.
 
-> Also start-wds works. wps uses **webpack-plugin-serve** while wds refers to **webpack-dev-server**.
+4. Ensure you have a `.storybook/preview.js|ts` (it can be empty)
```

Comment:
Maybe this should be optional? Can it work without?

Reply:
Yeah, I could put it in a try/catch for sure, let me do that.